web/skin/10h16/_js/spenden.wikimedia.js

```diff
@@ -171,21 +116,7 @@ $(function() {
       });
     }
 
-    $('#country').selectmenu({
-      change: function () {
-        var countryCode = 'generic';
-        if (countrySpecifics[$(this).val()]) {
-          countryCode = $(this).val();
-        }
-
-        $.each(countrySpecifics[countryCode], function (id, values) {
-          var $field = $('#' + id);
-          $.each(values, function (key, value) {
-            $field.attr(key, value);
-          });
-        });
-      }
-    });
+    $('#country').selectmenu({});
```

Comment:
This line is not necessary. All `<select />` are being initialized as a select menu a [few lines above](https://github.com/wmde/FundraisingFrontend/blob/3f7b75713920431a010a9ad96755b5693ecb7b92/web/skin/10h16/_js/spenden.wikimedia.js#L95-L106).